src/validations/rules/sar.sch

```diff
@@ -283,6 +287,7 @@
             <sch:assert
                 diagnostics="has-sap-rlink-diagnostic"
                 doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Results (SAR) §3.5"
+                fedramp:specific="true"
```

Comment:
This is specific to OSCAL (not FedRAMP). All of the inter-document elements are fundamental OSCAL constructs.